ci-operator/config/openshift/route-monitor-operator/openshift-route-monitor-operator-master.yaml

```diff
@@ -12,6 +12,9 @@ images:
 - dockerfile_path: Dockerfile
   from: base
   to: route-monitor-operator
+promotion:
+  name: "4.7"
```

Comment:
should this be 4.8?

Reply:
This promotion section isn't needed at all, and should be removed.